config/backup.php

```diff
@@ -147,16 +147,14 @@
         [
             'name' => config('app.name'),
             'disks' => ['local'],
-            'newest_backups_should_not_be_older_than_days' => 1,
-            'storage_used_may_not_be_higher_than_megabytes' => 5000,
+            'health_checks' => [],
```

Comment:
Add those three health checks here

Reply:
In fact `IsReachable` is prepended automatically in `BackupDestinationStatus` as it is a quite essential. No other checks makes sense if it is not reachable.

So I've just added the 2 other checks with their default config

Reply:
Perfect!